### PR TITLE
Lmsa 8657 rivet2

### PR DESCRIPTION
**Pull request by mrw-iu**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -164,11 +164,6 @@
             <artifactId>jquery</artifactId>
             <version>${jquery.version}</version>
         </dependency>
-        <dependency>
-            <groupId>org.webjars</groupId>
-            <artifactId>font-awesome</artifactId>
-            <version>${webjars-fontawesome.version}</version>
-        </dependency>
```
Comment (maurercw) [inline pom.xml:171]: There's a property up on line 82 that you can remove as well.

File: src/main/resources/static/js/loading.js [hunk 2]
```diff
@@ -41,10 +41,16 @@ function buttonLoading(button, formSubmit = true) {
         buttonsToDisable[i].disabled = true;
     }
     button.classList.add("rvt-button--loading");
-    button.getElementsByTagName('div')[0].classList.remove("hide");
+    button.getElementsByTagName('div')[0].classList.remove("rvt-display-none");
 
     if (formSubmit) {
         // FF doesn't need this, but Chrome and Edge do
-        button.form.submit();
```
Comment (mrw-iu) [inline src/main/resources/static/js/loading.js:48]: add form id as data attribute

File: src/main/resources/templates/index.html [hunk 3]
```diff
@@ -132,40 +130,37 @@ <h1 class="rvt-ts-32" th:text="#{blueprint.pageTitle}">Page title</h1>
...
-                                                        More text!
-                                                    </div>
-                                                </div>
-                                                <div th:id="|${mapEntry.key}Toggler|" class="hide" aria-hidden="true">
-                                                    <fieldset th:id="|${mapEntry.key}Checkboxes|">
-                                                        <legend class="sr-only" th:text="|Lock options for #{|type.${mapEntry.key}.title|}|"></legend>
-                                                        <ul class="blueprintNoStyleList"
-                                                            th:each="item : ${mapEntry.value}">
-                                                            <li>
+                                            <div class="rvt-accordion" data-rvt-accordion="test-accordion">
```
Comment (gjthomas) [inline src/main/resources/templates/index.html:144]: maybe a different name for the data-rvt-accordion

File: src/main/resources/templates/index.html [hunk 4]
```diff
@@ -187,55 +182,60 @@ <h1 class="rvt-ts-32" th:text="#{blueprint.pageTitle}">Page title</h1>
...
+                            </fieldset>
+                        </div>
+                        <div class="rvt-dialog__controls">
+                            <button id="completeSync" class="rvt-button" onclick="buttonLoading(this)" data-action="sync">
+                                <span class="rvt-button__content" th:text="#{dialog.complete}">Complete</span>
+                                <div class="rvt-loader rvt-loader--xs rvt-display-none" aria-label="Starting course sync"></div>
                             </button>
+                            <button id="cancelSync" type="button" class="rvt-button rvt-button--secondary" th:text="#{dialog.cancel}" data-rvt-dialog-close="dialog-sync-options">Cancel</button>
                         </div>
+                        <button id="closeDialog" class="rvt-button rvt-button--plain rvt-dialog__close" data-rvt-dialog-close="dialog-example" role="button">
```
Comment (gjthomas) [inline src/main/resources/templates/index.html]: different name for data-rvt-dialog-close

File: src/main/resources/templates/layout.html [hunk 5]
```diff
@@ -41,13 +41,13 @@
...
-    <link media="all" rel="stylesheet" type="text/css" th:href="@{/app/jsrivet/rivet-bundle.min.css}" />
+    <link media="all" rel="stylesheet" type="text/css" th:href="@{/app/jsrivet/rivet.min.css}" />
+    <link rel="stylesheet" type="text/css" th:href="@{/app/jsrivet/rivet-icons.css}"/>
     <link media="all" rel="stylesheet" type="text/css" th:href="@{/app/css/blueprint.css}"/>
 </head>
 <body>
 
-<div layout:fragment="content" class="rvt-container" role="main">
+<div layout:fragment="content" class="rvt-container-xl" role="main" aria-labelledby="blueprint-title">
     Page content goes here
```
Comment (mrw-iu) [inline src/main/resources/templates/layout.html:51]: blueprint-title consistent?

File: pom.xml [hunk 6]
```diff
@@ -73,7 +73,7 @@
         <lms-canvas-services.version>5.0.7</lms-canvas-services.version>
         <lms-canvas-common-configuration.version>5.0.3</lms-canvas-common-configuration.version>
         <lms-canvas-lti-framework.version>5.1.9</lms-canvas-lti-framework.version>
-        <lms-canvas-rivet.version>5.1.8.2_1</lms-canvas-rivet.version>
+        <lms-canvas-rivet.version>5.2.2.0</lms-canvas-rivet.version>
         <lms-canvas-redis-config.version>5.0.7</lms-canvas-redis-config.version>
```
Comment (mrw-iu) [inline pom.xml:77]: 5.2.2.0_1

**Pull request merged**